input/cql/SupplementalDataElements.cql

```diff
@@ -38,15 +48,22 @@ define "SDE Payer":
     }
 
 define "SDE Race":
-  Patient.race R
+  Patient.extension E
+    where E.url = 'http://hl7.org/fhir/us/core/StructureDefinition/us-core-race' 
     return Tuple {
-      codes: R.ombCategory union R.detailed,
-      display: R.text
+      codes: { 
+        E.extension e
+          where e.url = 'ombCategory'
+          return e.value as Code
+      } 
+      union {
+        E.extension e
+          where e.url = 'detailed'
+          return e.value as Code
+      },
+      display: E.value
```

Comment:
I think Patient.race is equivalent to Patient.extension[url = http://hl7.org/fhir/us/core/StructureDefinition/us-core-race], was there an explicit reason for this change?

I think both the "old" and the "new" implementations would result in the same value.

Reply:
The old implementation was showing an error

Reply:
Ultimately, the ICF team will be responsible for the SupplementalDataElements.cql uplift, so I think your workaround is appropriate.